January 12, 2016/talking-points.md

```diff
@@ -2,11 +2,11 @@
 
 ## Start of Class
```

Comment:
I should mention the "talking-points" file are notes for myself. I'm fine with them getting updated, but it's not necessarily for them to be covered in the TF groups.